content/about/limits.md

```diff
@@ -1,44 +1,78 @@
-﻿---
+---
 title: Limits
 weight: 4
 ---
 
+- [Plans](#plans)
+- [Script Size](#script-size)
+- [Number of Scripts](#number-of-scripts)
+- [Request Limits](#request-limits)
+  - [Burst Rate Limit](#burst-rate-limit)
+  - [Daily Request Limit](#daily-request-limit)
+     - [Fail Open](#fail-open)
+     - [Fail Closed](#fail-closed)
+- [CPU/Execution Time Limit](#cpu-execution-time-limit)
+- [Memory](#memory)
+- [Subrequests](#subrequests)
+- [Simultaneous Open Connections](#simultaneous-open-connections)
+  
 ## Plans
 
-Workers plans are separate from any Cloudflare plan (Free, Professional, Business) you may have. Everyone has access to the Free tier Workers platform by default. Upgrading to the Unlimited (Paid) plan will affect the number of requests your scripts can handle, as well as the compute time available per request.
+Workers plans are separate from any Cloudflare plan (Free, Professional, Business) you may have. Everyone has access to the Free tier Workers platform by default and are subject to the [free tier request limits](#request-limits). Upgrading to the Unlimited (Paid) plan lifts the restrictions and increases your CPU time limit.
```

Comment:
should free be capitalized here?

Reply:
we say "free tier" twice here, once with free capitalized and once without - these should be the same

Reply:
ok, updated